docs/3.1/docs/json-api/index.rst

```diff
@@ -3,87 +3,116 @@
 
 .. _json-api:
 
-HTTP JSON API Service
-#####################
+HTTP JSON API Service V2
+########################
 
-The **JSON API** provides a significantly simpler way to interact with a ledger than
-:doc:`the Ledger API </app-dev/ledger-api>` by providing *basic active contract set functionality*:
+This section describes the new JSON API Service that is a part of Canton 3.2. If you are looking for legacy JSON API, please navigate to :doc:`JSON API V1 </json-api/v1/index>`
```

Comment:
Actually I have no idea how to deal with that. I think we will not backport JSON V2 to 3.1. So that will be 3.2
On the other hand it probably makes no sense to create new branch of docs yet.